sample/src/main/java/com/yalantis/ucrop/sample/SampleActivity.java

```diff
@@ -168,6 +206,26 @@ public void afterTextChanged(Editable s) {
         }
     };
 
+    private TextWatcher mMaxSizeTextWatcher = new TextWatcher() {
```

Comment:
As an option you could create abstract class that extends TextWatcher, and then you could use it overriding only the afterTextChanged method